lib/config-loader.js

```diff
@@ -5,40 +5,20 @@ const path = require('path')
 const findUp = require('find-up')
 const stripJSONComments = require('strip-json-comments')
 
-let config = {}
 const configSources = ['package.json', '.rtlcssrc', '.rtlcss.json']
-const environments = [
-  process.env.USERPROFILE,
-  process.env.HOMEPATH,
-  process.env.HOME
-]
+const environments = [process.env.USERPROFILE, process.env.HOMEPATH, process.env.HOME]
 
 module.exports.load = (configFilePath, cwd, overrides) => {
   if (configFilePath) {
     return override(readConfig(configFilePath), overrides)
   }
 
-  const directory = cwd || process.cwd()
-  config = loadConfig(directory)
+  // Look first in cwd
+  environments.unshift(cwd || process.cwd())
```

Comment:
Maybe it'd be more clear if we did this instead?

```js
  const environments = [
    cwd || process.cwd(),
    process.env.USERPROFILE,
    process.env.HOMEPATH,
    process.env.HOME
  ]
```

This whole file is poorly covered by tests